src/components/_channel/ChannelCardBase/ChannelCardBase.tsx

```diff
@@ -37,36 +38,51 @@ export const ChannelCardBase: React.FC<ChannelCardBaseProps> = ({
   avatarUrl,
   isLoadingAvatar,
   isFollowing,
-  onFollow,
   className,
   onClick,
+  onFollow,
 }) => {
+  const mdMatch = useMediaMatch('md')
+  const [activeDisabled, setActiveDisabled] = useState(false)
+
+  const handleFollowButtonClick = (e: React.MouseEvent) => {
+    e.preventDefault()
+    onFollow?.(e)
+  }
   return (
-    <ChannelCardArticle className={className}>
+    <ChannelCardArticle className={className} activeDisabled={activeDisabled}>
       <ChannelCardAnchor onClick={onClick} to={id ? absoluteRoutes.viewer.channel(id) : ''}>
         <StyledAvatar size="channel-card" loading={isLoadingAvatar} assetUrl={avatarUrl} />
         <SwitchTransition>
           <CSSTransition
             key={isLoading ? 'placeholder' : 'content'}
-            timeout={parseInt(transitions.timings.sharp)}
+            timeout={parseInt(cVar('animationTransitionFast', true))}
             classNames={transitions.names.fade}
           >
             <InfoWrapper>
               {isLoading ? (
                 <>
-                  <SkeletonLoader width="100px" height="20px" bottomSpace="4px" />
-                  <SkeletonLoader width="70px" height="20px" bottomSpace="16px" />
-                  <SkeletonLoader width="60px" height="30px" />
+                  <SkeletonLoader width={100} height={mdMatch ? 24 : 20} bottomSpace={mdMatch ? 4 : 8} />
+                  <SkeletonLoader width={70} height={mdMatch ? 20 : 16} bottomSpace={onFollow && 16} />
+                  {onFollow && <SkeletonLoader width={60} height={32} />}
                 </>
               ) : (
                 <>
-                  <ChannelTitle variant="h300">{title}</ChannelTitle>
-                  <ChannelFollows variant="t200" secondary>
+                  <ChannelTitle variant={mdMatch ? 'h300' : 't200-strong'}>{title}</ChannelTitle>
+                  <ChannelFollows variant={mdMatch ? 't200' : 't100'} secondary>
                     {formatNumberShort(follows || 0)} followers
                   </ChannelFollows>
-                  <FollowButton variant="secondary" size="small" onClick={onFollow}>
-                    {isFollowing ? 'Unfollow' : 'Follow'}
-                  </FollowButton>
+                  {onFollow && (
+                    <FollowButton
+                      variant="secondary"
+                      size="small"
+                      onClick={handleFollowButtonClick}
+                      onMouseOut={() => setActiveDisabled(false)}
```

Comment:
Does `stopPropagation` wouldn't do the job?

Reply:
Unfortunately, it doesn't - clicking the follow button triggers pulldown animation and my intention was to not do that. There is a CSS-only solution for this, but it's not supported on Safari(requires using `focus-within`). We discussed this on RocketChat when I was working on VideoTile.